app/assets/javascripts/samples/sample_datatable.js.erb

```diff
@@ -1193,34 +1193,56 @@ function changeToEditMode() {
 
       cancelEditMode();
 
-      var self = $(this);
-      var li = self.closest('li');
-      var text = li.find('.text');
-      if ($(text).find('.modal-tooltiptext').length > 0) {
-        text = $(text).find('.modal-tooltiptext');
+      var li = $(this).closest('li');
+      var url = li.attr('data-edit-url') + "/electronic_sign_edit";
```

Comment:
please don't hard code urls. Here you have erb file so you can pass <%= link_to(whatever) %> in there